### ISSUE TITLE
Add Benchmarks to Readme

### ISSUE THREAD

**Issue by ekryski** [JerrySievert/bricks#1]
Hey Jerry,

Looks pretty good.  I think it would be good to include the benchmarks in the readme that we had discussed last week.

**Comment by JerrySievert** [comment]
Yeah, I was rushing to the finish line before the beginning of the week :)
